docker-compose.yml

```diff
@@ -23,11 +36,15 @@ services:
       - go-cache:/go/pkg
     working_dir: /go/src
     depends_on:
-      - service
+      - minio
       - postgres
+      - service
     environment:
-      FML_DATABASE_URI: postgres://postgres:postgres@postgres/postgres
+      S3_BASE_URL: http://minio:9000
```

Comment:
could we use `FML_S3_ENDPOINT_URI` for both the app and the integration tests?

Reply:
ok. fixed.